app/assets/v2/js/pages/new_bounty.js

```diff
@@ -817,19 +833,35 @@ $('#submitBounty').validate({
           deductBountyAmount(fee, '');
         } else {
           if (isETH) {
-            web3.eth.sendTransaction({
+            const params = {
               to: to_address,
               from: web3.eth.coinbase,
               value: web3.toWei(fee, 'ether'),
               gasPrice: gas_price
-            }, function(error, txnId) {
-              indicateMetamaskPopup(true);
-              if (error) {
-                _alert({ message: gettext('Unable to pay bounty fee. Please try again.') }, 'error');
-              } else {
-                deductBountyAmount(fee, txnId);
-              }
-            });
+            };
+
+            window.isWeb3Fired = true;
+            if (web3.currentProvider.isFortmatic) {
+              const batch = web3.createBatch();
```

Comment:
Fortmatic doesn't support concurrent transactions, so they have to be batched. I only do it when the provider is Fortmatic, but I think it would be safe to always do it, I suppose MetaMask should handle it correctly.